ComponentKit/DataSources/CKCollectionViewTransactionalDataSource.mm

```diff
@@ -61,9 +61,14 @@ - (void)applyChangeset:(CKTransactionalComponentDataSourceChangeset *)changeset
                               userInfo:userInfo];
 }
 
-static void applyChangesToCollectionView(CKTransactionalComponentDataSourceAppliedChanges *changes, UICollectionView *collectionView)
+static void applyChangesToCollectionView(CKTransactionalComponentDataSourceAppliedChanges *changes, UICollectionView *collectionView, CKTransactionalComponentDataSourceState *currentState, CKComponentDataSourceAttachController *attachController)
 {
-  [collectionView reloadItemsAtIndexPaths:[changes.updatedIndexPaths allObjects]];
+  NSMutableArray *reloadIndexPaths = @[].mutableCopy;
```

Comment:
This is a bit inefficient as it incurs a full copy of the array, can we do `[NSMutableArray array]` instead ?